src/renderer/components/dialogs/Settings.js

```diff
@@ -93,26 +103,37 @@ class Settings extends React.Component {
     this.props.userFeedback({ type: 'success', text: 'Account updated successfully' })
   }
 
+  onAdvancedSettingsOpen () {
+    ipcRenderer.send('retrieveAdvancedSettingsObj')
+  }
+
+  retrieveAdvancedSettingsObjResp (e, advancedSettingsObj) {
+    console.log('retrieveAdvancedSettingsObjResp', advancedSettingsObj)
+    this.setState({ advancedSettings: advancedSettingsObj })
```

Comment:
And do `this.setState({ advancedSettings })`

Reply:
+1